tests/utils.py

```diff
@@ -142,7 +142,7 @@ def make_venv(venv_dir: Path) -> VenvInfo:
     try:
         venv.create(venv_dir, with_pip=True, clear=True)
     except subprocess.CalledProcessError as e:
-        if "ensurepip" in e.cmd:
+        if "ensurepip" in e.cmd and "KeyboardInterrupt" not in e.stdout:
```

Comment:
The "if you're on Linux, you might need to install the python3-venv package" message was showing up every time I interrupted the script with a `KeyboardInterrupt` during the "`pip install` requirements into the venvs" step, and it was kinda annoying.